README.md

```diff
@@ -63,7 +112,178 @@ And finally you can start the VM:
+ Code                | [charges-deed-api](https://github.com/LandRegistry/charges-deed-api)
+ Infrastructure code | [charges-deed-api/puppet/deed_api](https://github.com/LandRegistry/charges-deed-api/tree/master/puppet/deed_api)
+ Development Host    | http://deed_api.dev.service.gov.uk
+ Code on VM          | /opt/deed_api
+ Default port        | 0.0.0.0:9012
+
+### Case API
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | case_api
+ Code                | [charges-case-api](https://github.com/LandRegistry/charges-case-api)
+ Infrastructure code | [charges-case-api/puppet/case_api](https://github.com/LandRegistry/charges-case-api/tree/master/puppet/case_api)
+ Development Host    | http://case_api.dev.service.gov.uk
+ Code on VM          | /opt/case_api
+ Default port        | 0.0.0.0:9070
+
+### Scribe
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | scribe
+ Code                | [charges-scribe](https://github.com/LandRegistry/charges-scribe)
+ Infrastructure code | [charges-scribe/puppet/scribe](https://github.com/LandRegistry/charges-scribe/tree/master/puppet/scribe)
+ Development Host    | http://scribe.dev.service.gov.uk
+ Code on VM          | /opt/scribe
+ Default port        | 0.0.0.0:9010
+
+### Verifudge
+
+ Piece               | Location
+---------------------|------------------------------------------------------------------------------------
+ App name            | verifudge
+ Code                | [charges-verifudge](https://github.com/LandRegistry/charges-verifudge)
+ Infrastructure code | [charges-verifudge/puppet/verifudge](https://github.com/LandRegistry/charges-verifudge/tree/master/puppet/verifudge)
+ Development Host    | http://verifudge.dev.service.gov.uk
+ Code on VM          | /opt/verifudge
+ Default port        | 0.0.0.0:9080
+
+##Development usage
+
+### Development path
+
+The VM uses rsync to mirror the path you set when installing:
+```bash
+export development=/path/to/your/code
+```
+
+This folder will exist in `~/development`. In this way you can keep using your
+IDE or editor of choice while benefitting from the Dev VM.
+
+### Controlling apps
+
+Every app is controlled through systemd. They each expose a service that can be
+used to do a variety of tasks. They also expose an nginx config that can be used
+in development to speed up the connecting of services together.
+
+The following instructions assume you have sshed in to a vm built using a clean
+copy of this repo.
+
+#### Starting / Stopping / Restarting an app
+
+To start / stop / restart any app use:
+
+```
+> sudo systemctl <action> -u <app_name>
+```
+
+Where <app_name> is the name listed in the [Apps installed](#apps-installed)
+tables above and action is any of `start`, `stop`, `restart`.
+
```

Comment:
Also what about mentioning accessing the logging and such using journctl -u <app_name> is it? And also to check status of an app using service <app_name> status?